sdk/core/src/lib.rs

```diff
@@ -40,6 +40,7 @@ use uuid::Uuid;
 pub use bytes_stream::*;
 pub use constants::*;
 pub use context::Context;
+pub use error::Result;
```

Comment:
Why just `Result` and not `errror::*`?

Reply:
We could do that though I think I'd leave that for a future PR.